app/components/AlertDialog/__tests__/AlertDialog.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import AlertDialog from '..';
+
+jest.mock('@/services/api');
+
+describe('AlertDialog', () => {
+  let component = render(
```

Comment:
this would be better inside the test. it makes it clearer what test this is for, and it's only used by the one test anyway